sim/shaman/earth_shock.go

```diff
@@ -54,8 +54,10 @@ func (shaman *Shaman) newEarthShockSpellConfig(rank int, shockTimer *core.Timer)
 	spell.ThreatMultiplier = 2
 	spell.BonusCoefficient = spellCoeff
 
+	bonusDamage := core.TernaryFloat64(shaman.Ranged().ID == TotemOfRage, 30*spellCoeff, 0)
```

Comment:
Can you add the bonus damage up above to the `baseDamageLow` and `baseDamageHigh` variables? It should have the same effect but will require 1 less calculation per cast. Same for frost shock